README.md

```diff
@@ -1,111 +1,39 @@
 # Datadog Fork
 
-This is an experimental fork of [elastic/otel-profiling-agent](https://github.com/elastic/otel-profiling-agent). The upstream project is in the process of being [donated](https://github.com/open-telemetry/community/issues/1918) to the OpenTelemetry project. Please refer to our [documentation](https://docs.datadoghq.com/profiler/) for a list of offically supported Datadog profilers.
+This is an experimental fork of [elastic/otel-profiling-agent](https://github.com/elastic/otel-profiling-agent). The upstream project is in the process of being [donated](https://github.com/open-telemetry/community/issues/1918) to the OpenTelemetry project. Please refer to our [documentation](https://docs.datadoghq.com/profiler/) for a list of officially supported Datadog profilers.
```

Comment:
We should probably update this section to indicate that we're now a fork of `open-telemetry/opentelemetry-ebpf-profiler` (if not in a technical sense, at least in spirit?). There is no more need to refer to the elastic donation, since that has completed.